src/certificates/empty-certificates-with-modes/EmptyCertificatesWithModes.jsx

```diff
@@ -0,0 +1,35 @@
+import { useDispatch } from 'react-redux';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button, Stack, Card } from '@openedx/paragon';
+import { Add as AddIcon } from '@openedx/paragon/icons';
+
+import { setMode } from '../data/slice';
+import { MODE_STATES } from '../data/constants';
+import messages from '../messages';
+
+const EmptyCertificatesWithModes = () => {
+  const intl = useIntl();
+  const dispatch = useDispatch();
+
+  const handleCreateMode = () => {
+    dispatch(setMode(MODE_STATES.create));
+  };
+
+  return (
+    <Card>
+      <Card.Section>
+        <Stack direction="horizontal" className="justify-content-center align-items-center" gap="3.5">
+          <span className="small">{intl.formatMessage(messages.noCertificatesText)}</span>
+          <Button
+            iconBefore={AddIcon}
+            onClick={handleCreateMode}
+          >
+            {intl.formatMessage(messages.setupCertificateBtn)}
+          </Button>
```

Comment:
It makes more sense for this to be use `ActionRow` instead of `Stack` and `ActionRow.Spacer` to separate the text and button.